helix-term/src/commands.rs

```diff
@@ -3415,6 +3415,10 @@ enum Paste {
     After,
 }
 
+// Only compiled once.
+#[allow(clippy::trivial_regex)]
+static PASTE_LINE_ENDING_REGEX: Lazy<Regex> = Lazy::new(|| Regex::new(r"\r\n|\r|\n").unwrap());
```

Comment:
Call it REGEX and move it inside the fn. Rust allows us to nest and scope arbitrarily so that way the static value is only visible within the fn.

Reply:
https://github.com/helix-editor/helix/blob/94abc52b3b0929f399cea14e1efcf2c1d0a31ad8/helix-term/src/commands.rs#L1056-L1065